src/types.ts

```diff
@@ -62,10 +63,22 @@ export interface ExporterConfig {
 	/**
 	 * The number of times the exporter should retry before returning failure (default: 3)
 	 */
-	maxRetries? : number;
+	maxRetries?: number;
 
 	/**
 	 * The time in milliseconds to wait before retrying an export that failed due to a connection error (default: 1000)
 	 */
-	retryDelay? : number;
+	retryDelay?: number;
+}
+
+export interface ReaderConfig {
+	/**
+	 * The interval in which metrics are exported using the PeriodicExportingMetricReader (default: 60000)
```

Comment:
Should we use the `@default` JSDoc tag for the default values for these properties?